_posts/2021-02-18-survey/survey.Rmd

```diff
@@ -0,0 +1,314 @@
+
+For [`tensorflow`](https://tensorflow.rstudio.com/), the website has a multitude of guides, tutorials, and examples. For [`torch`](torch.mlverse.org), reflecting the discrepancy in respective lifecycles, materials are not that abundant (yet). However, after a recent refactoring, the website has a new, four-part [Get started](https://torch.mlverse.org/start/) section addressed to both beginners in DL and experienced TensorFlow users curious to learn about `torch`. After this hands-on introduction, a good place to get more technical background would be the section on [tensors, autograd, and neural network modules](https://torch.mlverse.org/technical/).
+
+Truth be told, though, nothing would be more helpful here than contributions from the community. Whenever you solve even the tiniest (which is often how things appear to oneself) problem, consider creating a vignette explaining what you did. Future users will be thankful, and a growing user base means that over time, it'll be your turn to find that some things have already been solved for you!
+
+Which leads me to the last -- aggregate -- point.
+
+##### Community, community, community
+
+The remaining things I'd like to mention didn't come up quite as often (individually), but taken together, they all have something in common: They all are wishes *we* happen to have, as well!
+
+This definitely holds in the abstract -- let me cite:
+
+> "Develop more of a DL community"
+>
+> "Larger developer community and ecosystem. Rstudio has made great tools, but for applied work is has been hard to work against the momentum of working in Python."
+
+We wholeheartedly agree: And building a larger community is exactly what we're trying to do. I like the formulation "a DL community" insofar it is framework-independent. In the end, frameworks are just tools, and what counts is our ability to usefully apply those tools to problems we need to solve.
+
+Concrete wishes include
+
+-   More paper/model implementations (such as TabNet).
+
+-   Facilities for easy data reshaping and pre-processing (e.g., in order to pass data to RNNs or 1dd convnets in the expected 3-d format).
+
+-   Probabilistic programming for `torch` (analogously to TensorFlow Probability)
+
+-   A high-level library (such as fast.ai) based on `torch.`
+
+If time and resources were infinite, all of these would have become reality already.
+
+But then, instead of having everything implemented by a small group of people, isn't it much more effective (and satisfying!) to have a large and diverse group of people each contribute "theirs": the things they're interested in most, or which they feel drawn to most due to their individual backgrounds? In other words, if you're motivated to do so, please feel very encouraged to contribute to the growing community!
+
+And now, on to Spark.
```

Comment:
```suggestion
```

Reply:
We know you've moved on to Spark because it's the next header. 😜

Reply:
uh-oh, all the chattiness being removed ;-)